.changeset/shy-carpets-own.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-org': patch
+---
+
+Added `entityLimit` to change the limit of entities displayed in the ownership boxes.
```

Comment:
As I mentioned previously it would be great to see an example of this here. Again, this gets turned into release notes which make upgrading Backstage easier. Being able to see this there means more people will use this feature or at least know about it.

Reply:
I would also include a link to the Storybook entry that shows examples of this: https://backstage.io/storybook/?path=/story/plugins-org-ownership-card--default

Reply:
Added example. Also attached storybook video.

Reply:
Sorry, this changeset still is just a single line. I don't see the example here?

Reply:
How to add example here ? As far as I understand, storybook example I've created. And also attached video for reference in PR description.
From where I can get link to this new story I created in storybook ?
Can you pls check attached video once ?

Reply:
Can you check this ? Added link to storybook example and also attached video in PR description

Reply:
All I was looking for was to just add something like this: `<OwnershipCard entityLimit={entityLimit} />` directly into the changeset but the link to the Storybook will work just as well 👍